src/identity/email-password/EmailPasswordIdentityProviderHandler.ts

```diff
@@ -2,21 +2,21 @@ import type { IncomingMessage, ServerResponse } from 'http';
 import { getLoggerFor } from '../../logging/LogUtil';
 import { HttpHandler } from '../../server/HttpHandler';
 import type { Guarded } from '../../util/GuardedStream';
-import type { OidcProviderFactory } from '../provider/OidcProviderFactory';
+import type { SolidIdentityProviderFactory } from '../provider/SolidIdentityProviderFactory';
 
 interface EmailPasswordIdentityProviderHandlerArgs {
-  oidcProviderFactory: OidcProviderFactory;
+  solidIdentityProviderFactory: SolidIdentityProviderFactory;
```

Comment:
I'm not sure that I understand the reasoning behind the name changes. A generic OIDCProviderFactory should be what is passed in. Not specifically a SolidIdentityProvider factory.